sdk/core/Azure.Core/samples/Diagnostics.md

```diff
@@ -164,21 +164,27 @@ If your are using Azure SDK libraries in ASP.NET Core application consider using
 
 ## ActivitySource support
 
-Azure SDKs released after October 2021 include experimental support for ActivitySource, which is a simplified way to create and listen to activities added in .NET 5.
+Azure SDKs support for `ActivitySource`, which is a simplified way to create and listen to activities which is compatible with OpenTelemetry.
 Azure SDKs produce the following kinds of Activities:
 
 - *HTTP calls*: every HTTP call originating from Azure SDKs
 - *client method calls*: for example, `BlobClient.DownloadTo` or `SecretClient.StartDeleteSecret`.
 - *messaging events*: Event Hubs and Service Bus message creation is traced and correlated with its sending, receiving, and processing.
 
-Because `ActivitySource` support is experimental, the shape of Activities may change in the future without notice.  This includes:
+Prior to November 2023, OpenTelemetry support was experimental for all Azure SDKs (see [Enabling experimental tracing features](#enabling-experimental-tracing-features) for the details). 
+Most of Azure SDKs released after December 2023 have OpenTelemetry support enabled by default, however tracing support remains experimental for messaging libraries (`Azure.Messaging.ServiceBus` and `Azure.Messaging.EventHubs`). 
```

Comment:
❤️ TY for this explanation!!

One nit: should we say something like "in or after" Nov 2023 here instead of December?